2-mandatory/2-financial-times.js

```diff
@@ -5,33 +5,69 @@
     Implement the function below, which will return a new array containing only article titles which will fit.
 */
 function potentialHeadlines(allArticleTitles) {
-    // TODO
+    const ARTICLE_TITLES = [];
+    for (let i = 0; i < allArticleTitles.length; i++) {
+       if (allArticleTitles[i].length <= 65) {
+        ARTICLE_TITLES.push(allArticleTitles[i]);
+       };
+       
+
+    }
+
+    return ARTICLE_TITLES;
 }
 
+
 /*
     The editor of the FT likes short headlines with only a few words!
     Implement the function below, which returns the title with the fewest words.
-    (you can assume words will always be seperated by a space)
+    (you can assume words will always be separated by a space)
 */
 function titleWithFewestWords(allArticleTitles) {
```

Comment:
This is a good attempt. I think there is one small problem with this code.
We want to look for the article with the fewest *words*. Have a think about what `.split("")` does in your code.